tests/ssh/test_ssh_stress.py

```diff
@@ -0,0 +1,221 @@
+import socket
+import threading
+import paramiko
+import time
+import pytest
+import logging
+from tests.common.helpers.assertions import pytest_assert
+
+pytestmark = [
+    pytest.mark.disable_loganalyzer,
+    pytest.mark.topology("any"),
+    pytest.mark.device_type("vs"),
+]
+
+START_BGP_NBRS = "sudo config bgp startup all"
+STOP_BGP_NBRS = "sudo config bgp shutdown all"
+
+SHUTDOWN_INTERFACE = "sudo config interface shutdown PortChannel0001"
+STARTUP_INTERFACE = "sudo config interface startup PortChannel0001"
+
+CONFIGURE_ACL = "acl-loader update full /tmp/acl.json"
+REMOVE_ACL = "acl-loader delete"
+
+ADD_ROUTE = "sudo config route add prefix 2.2.3.4/32 nexthop vrf Vrf-RED 30.0.0.9"
+REMOVE_ROUTE = "sudo config route del prefix 2.2.3.4/32 nexthop vrf Vrf-RED 30.0.0.9"
+
+ADD_PORTCHANNEL = "sudo config portchannel add PortChannel0010"
+REMOVE_PORTCHANNEL = "sudo config portchannel del PortChannel0010"
+
+done = False
+max_cpu = 0
+max_mem = 0
+
+@pytest.fixture
+def setup_teardown(duthosts, rand_one_dut_hostname):
+    duthost = duthosts[rand_one_dut_hostname]
+
+    # Copies over ACL configs for the ACL commands
+    duthost.copy(src="acl/templates/acltb_test_rules.j2", dest="/tmp/acl.json", mode="0755")
+
+    yield
+
+    duthost.file(path="/tmp/acl.json", state="absent")
+
+    duthost.shell_cmds(cmds=[START_BGP_NBRS, STARTUP_INTERFACE, REMOVE_ACL, REMOVE_ROUTE, REMOVE_PORTCHANNEL], module_ignore_errors=True)
+
+
+def get_system_stats(duthost):
+    """Gets Memory and CPU usage from DUT"""
+    stdout_lines = duthost.command("vmstat")["stdout_lines"]
+    data = list(map(float, stdout_lines[2].split()))
+
+    total_memory  = sum(data[2:6])
+    used_memory = sum(data[4:6])
+
+    total_cpu = sum(data[12:15])
+    used_cpu = sum(data[12:14])
+
+    return used_memory/total_memory, used_cpu/total_cpu
+
+def start_SSH_connection(dut_mgmt_ip):
+    """Starts SSH connection to provided IP"""
+    ssh = paramiko.SSHClient()
+    ssh.set_missing_host_key_policy(paramiko.AutoAddPolicy())
+    ssh.connect(dut_mgmt_ip, username="admin", password="password", allow_agent=False, look_for_keys=False)
+
+    return ssh
+
+def monitor_system(duthost):
+    """Monitors system memory and CPU for duration of test"""
+    global max_mem, max_cpu
+
+    while not done:
+        dut_stats = get_system_stats(duthost)
+        logging.info("Memory Usage: {}% | CPU Usage: {}%".format(dut_stats[0]*100, dut_stats[1]*100))
+
+        max_mem = max(max_mem, dut_stats[0])
+        max_cpu = max(max_cpu, dut_stats[1])
```

Comment:
Is this part of the 'stress'?  If not, can you add a sleep in the loop? This loop will generate lots of logs. Do these logs needs to stay at info level or should they be at debug level?

Reply:
please remove some blank lines here. The rest of the file only have 1 blank line between functions.